cpp/velox/compute/VeloxPlanConverter.cc

```diff
@@ -221,24 +224,44 @@ std::shared_ptr<gluten::RecordBatchResultIterator>
 VeloxPlanConverter::GetResultIterator() {
   std::shared_ptr<gluten::RecordBatchResultIterator> resIter;
   const std::shared_ptr<const core::PlanNode> planNode = getVeloxPlanNode(plan_);
+
+  auto splitInfos = subVeloxPlanConverter_->splitInfos();
+  auto leafPlanNodeIds = planNode->leafPlanNodeIds();
+  // Here only one leaf node is expected here.
+  assert(leafPlanNodeIds.size() == 1);
+  auto iter = leafPlanNodeIds.begin();
+  auto splitInfo = splitInfos[*iter].get();
+
+  // Get the information for TableScan.
+  u_int32_t partitionIndex = splitInfo->partitionIndex;
+  const auto& paths = splitInfo->paths;
+  const auto& starts = splitInfo->starts;
+  const auto& lengths = splitInfo->lengths;
+  const auto format = splitInfo->format;
```

Comment:
Will DWRF and ORC share the same kind of split?
```
splitInfo->format = dwio::common::FileFormat::ORC;
```

Reply:
Sure. It seems velox use the same dwrf reader and dwrf writer to handle the orc and dwrf file.